recipe/patches/0001-0.10.0-palig.patch

```diff
@@ -0,0 +1,3230 @@
+ 
+ The above string is understandable by a C++11 compiler, but it quickly reaches the limit of our developer brain, so most of the time, we are more comfortable with the Python backend::
+ 
+diff --git a/docs/conf.py b/docs/conf.py
+index 79528ee2a..983f7bfe4 100644
+--- a/docs/conf.py
++++ b/docs/conf.py
+@@ -38,7 +38,7 @@ with open("../README.rst") as readme:
+     Changelog
+ '''
+ 
+-    readme_body = readme_body.replace('http://pythran.readthedocs.io', toc)
++    readme_body = readme_body.replace('https://pythran.readthedocs.io', toc)
+ 
+     with open("index.rst", "w") as index:
+         index.write(readme_body)
+diff --git a/omp/__init__.py b/omp/__init__.py
+index bddae3063..349689105 100644
+--- a/omp/__init__.py
++++ b/omp/__init__.py
+@@ -44,42 +44,34 @@ class OpenMP(object):
+             self.version = 20
+ 
+     def get_libomp_names(self):
+-        """Return list of OpenMP libraries to try, based on platform and
+-        compiler detected."""
+-        if cxx is None:
+-            # Can't tell what compiler we're using, guessing we need libgomp
+-            names = ['libgomp']
+-        else:
+-            cmd = [cxx, '--version']
+-            try:
+-                version_str = os.path.dirname(check_output(cmd).decode().strip())
+-            except (OSError, CalledProcessError):
+-                version_str = ''
+-
+-            if 'clang' in version_str:
+-                names = ['libomp', 'libiomp5', 'libgomp']
+-            elif version_str.startswith('Intel'):
+-                names = ['libiomp5']
+-            else:
+-                # Too many GCC flavors and version strings, make this the default
+-                # rather than try to detect if it's GCC
+-                names = ['libgomp']
+-
++        """Return list of OpenMP libraries to try"""
++        names = ['omp', 'gomp', 'iomp5']
+         return [name + get_shared_lib_extension() for name in names]
```

Comment:
you need to remove the shared lib extension too

Reply:
ok, and use the extension only when needed.